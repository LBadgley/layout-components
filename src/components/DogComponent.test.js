import React from 'react';
import { shallow } from 'enzyme';
import Dog from './DogComponent';

describe('dog component tests', () => {
  it('renders a dog with stats', () => {
    const wrapper = shallow(<Dog />);
    expect(wrapper).toMatchSnapshot();
  });
});
