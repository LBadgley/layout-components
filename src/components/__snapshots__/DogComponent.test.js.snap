// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dog component tests renders a dog with stats 1`] = `
<dl>
  <dt>
    Name:
  </dt>
  <dd>
    Ruby
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    2
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    7.5 lbs
  </dd>
</dl>
`;
